narwhals/dataframe.py

```diff
@@ -2666,42 +2676,39 @@ def null_count(self: Self) -> Self:
             A dataframe of shape (1, n_columns).
 
         Notes:
-            pandas and Polars handle null values differently. Polars distinguishes
-            between NaN and Null, whereas pandas doesn't.
+            pandas handles null values differently from Polars and PyArrow.
+            See https://narwhals-dev.github.io/narwhals/pandas_like_concepts/null_handling/
+            for reference.
 
```

Comment:
Same here about the link `[null_handling](https://narwhals-dev.github.io/narwhals/pandas_like_concepts/null_handling)` ... ?